app/src/main/java/com/example/ncov19traking/ui/CovidCustomCard.kt

```diff
@@ -0,0 +1,86 @@
+package com.example.ncov19traking.ui
+
+import android.content.Context
+import android.graphics.Color
```

Comment:
this is not being used.

Reply:
I forgot configuring automatic optimize import